fast_api_als/services/verify_phone_and_email.py

```diff
@@ -13,6 +13,10 @@
 You also trying to undderstand the execution time factor.
 """
 
+logging.basicConfig(filename='verify_phone_and_email.log', format="%(levelname)s: %(message)s", level=logging.DEBUG)
```

Comment:
no need to add this in each file. You can create a separate util for your custom logger and import that module where you require it

Reply:
for seperating logs into different files, I tried to do it, can it still be done the other way?

Reply:
simplest way is to define a basicConfig in main.py and just use getLogger in each module to specify the module name for the log viewer.

Reply:
use `logger = logging.getLogger(__name__)` in different modules to use a module-level logger